client/src/pages/GridTemplateContainer.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { makeStyles, Grid } from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1
+  }
+}));
+
+function GridTemplateContainer(props) {
```

Comment:
Do you think we should just combine GridTemplateContainer, and LoginSignupContainer, given this is so small - you can also pass in variables to these containers with default values to make this container editable

Reply:
You can ignore this comment, I just noticed that Stanley uses this also

Reply:
Although, actually I feel like GridTemplateContainer should be used inside LoginSignupContainer to avoid repeated code